tests/static/all_question_types.tsv

```diff
@@ -28,6 +28,6 @@ Question	Type	Text (en)	Text (hin)	Audio (en)	Audio (hin)	Image (en)	Image (hin)
 /question22/question23/question28	Password	question28	question28									no
 /question22/question23/question29	Signature Capture	question29	question29									no
 /question22/question23/question7	Android App Callout	question7	question7									no
-	Base											no
 /question20	Hidden Value											no
-/question32	Hidden Value										1 + 2	no
\ No newline at end of file
+/question32	Hidden Value										1 + 2	no
+	Base											no
```

Comment:
I think it's because custom xml is turned into a ReadOnly Mug that can't be edited in Vellum, but still shows up. I'm not sure of it's use here, but didn't really investigate further